app.py

```diff
@@ -0,0 +1,172 @@
+import logging

```

Comment:
Please also add a link to this file with the tutorial code you used

Reply:
added! 

#starting code from https://github.com/slackapi/python-slack-sdk/tree/main/tutorial